package.json

```diff
@@ -127,5 +127,8 @@
     "lint:js",
     "lint:style"
   ],
-  "private": true
+  "private": true,
+  "dependencies": {
+    "ember-cli-file-saver": "^1.2.2"
```

Comment:
I think this should go into dev-dependencies (for.... ember reasons). I'm guessing you installed with `yarn`, instead do `ember install ember-cli-file-saver` and that will put things where they go.  Past experience also indicates that you will have to then remove and empty `dependencies` section in `package.json`